.yamato/z_preview-publish.yml

```diff
@@ -72,33 +72,45 @@ publish_{{ test.name }}:
     packages:
         paths:
             - "upm-ci~/packages/*.tgz"
-{% if publishing.auto_publish == "true" -%}
-  triggers:
-    recurring:
-      - branch: "{{ repo.branches.integration }}"
-        frequency: daily
-{% endif -%}
   dependencies:
-    - .yamato/upm-ci-abv.yml#all_project_ci_nightly_trunk
+    - .yamato/z_preview-publish.yml#wait_for_nightly
 {% for testing in extra_jobs.testing -%}
     - {{ testing }}
 {% endfor -%}
     - .yamato/upm-ci-packages.yml#pack_{{ test.name }}
 {% for editor in editors -%}
 {% for platform in platforms -%}
     - .yamato/upm-ci-packages.yml#test_{{ test.name }}_{{ platform.os }}_{{ editor.version }}
```

Comment:
You can remove this dependency from here since the wait_for_nightly already has that as a dependency.
An optional change, it will just reduce the number of lines between jobs in the pipeline

Reply:
removed in latest commit